script/constructron.lua

```diff
@@ -922,25 +883,25 @@ me.conditions = {
     ---@param _ any
     ---@return boolean | string
     upgrade_done = function(job, _, _, _)

```

Comment:
the return value annotation seems to be incorrect now.
```suggestion
    ---@param _ any
    ---@return boolean?
    upgrade_done = function(job, _, _, _)
```